src/FormCollection/Form/UXCollectionEntryToolbarType.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Symfony\UX\FormCollection\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ButtonType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @final
+ * @experimental
+ */
+class UXCollectionEntryToolbarType extends AbstractType
+{
```

Comment:
The `UXCollectionEntryToolbarType` and `UXCollectionToolbarType` allows to add additional buttons later via a form extension. Example can be a `Copy & Paste` Functionality.